package.json

```diff
@@ -4,14 +4,15 @@
   "description": "RxJS powered Redux for Angular2 apps",
   "main": "./index.js",
   "scripts": {
-    "test": "npm run build_test && jasmine",
-    "clean_test": "rm -rf tmp/",
-    "build_test": "npm run clean_test && tsc -p spec",
-    "build_dist": "rm -rf dist && tsc && npm run copy_package",
-    "copy_package": "cp package.json dist/package.json",
-    "_prepublish": "npm run typings && npm run build_dist",
-    "lint": "tslint src/**.ts",
-    "typings": "typings install"
+    "clean": "./node_modules/.bin/rimraf node_modules typings dist tmp npm-debug.log && npm cache clean",
```

Comment:
All of these are unnecessary. When running scripts, NPM automatically puts ./node_modules/.bin/ in the "path" so that locally installed tools will be found there. The tools do not need to be installed globally and you do not need to say ./node_modules/.bin/ in scripts.